topics/RDMA.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
```

Comment:
There is no mention of the special make that is required to generate the `valkey-rdma.so`.  Is that covered somewhere else?  If so, a link to that would help a sys-admin that is trying to get started.

Reply:
> There is no mention of the special make that is required to generate the `valkey-rdma.so`. Is that covered somewhere else? If so, a link to that would help a sys-admin that is trying to get started.
> 

Yes, this part is in valkey/README.md. Because the code is not merged into valkey source now, I think I can put the link after code gets merged.

> Are kernel mods still required? Does the sys-admin need to install any thing like libraries or the service? If so, this seems like the best place to say so.
> 

I guess setup RDMA env (both hardware and software) is not part of this document, before using RDMA, a user should know if there is any.

BTW, I also notice libibverbs and librdmacm are auto installed by some Linux OS.

> There are some hints in this direction in the "How does Valkey use RDMA" section. Perhaps they should be moved to "getting started" and be spelt out in detail.

Reply:
> > Are kernel mods still required? Does the sys-admin need to install any thing like libraries or the service? If so, this seems like the best place to say so.
> 
> I guess setup RDMA env (both hardware and software) is not part of this document, before using RDMA, a user should know if there is any.
> 
> BTW, I also notice libibverbs and librdmacm are auto installed by some Linux OS.

This document doesn't even hint at the prerequisites for Valkey Over RDMA.  Yes, someone that already knows about RDMA will be fine.  However, someone that does not already know about RDMA could come to this document and say "Hey, this looks cool, 2.5 times faster, why doesn't everyone use it?  Let's try it out."  At the very least, I think it would be polite to provide such a person a hint that they will need special hardware and software set up, and where to learn more.  I looked at the link to the Wikipeadia page that you included and was surprised that it never once stated that you need a special kind of network card.

Maybe just "Before using Valkey over RDMA, you must understand how to configure RDMA on your host and client machines."